src/styles/blocks/main-page.scss

```diff
@@ -0,0 +1,13 @@
+.main-page {
+  font-family: "Roboto", sans-serif;

```

Comment:
```suggestion
  font-family: "Roboto", sans-serif;

```

Reply:
would be better to move to html selector.

Reply:
I had a such fix, so recommended moving it)

Reply:
It is ok to style `html` by tag selector